phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -2127,6 +2124,13 @@ public MetaDataResponse call(MetaDataService instance) throws IOException {
                     }
                 }
             }
+            else if (result.getMutationCode() == MutationCode.UNALLOWED_TABLE_MUTATION) {
```

Comment:
nit : put it on the same line as closing bracket above.

Reply:
We can combine this check and the bottom `if` condition:

```java
if (modifyHTable && result.getMutationCode() != MutationCode.UNALLOWED_TABLE_MUTATION) {
    sendHBaseMetaData(tableDescriptors, pollingNeeded);
}
```